packages/firebase/package.json

```diff
@@ -227,6 +227,18 @@
       },
       "default": "./storage/dist/esm/index.esm.js"
     },
+    "./ai": {
```

Comment:
Will also need an actual physical directory under `packages/firebase` with a `package.json` and a `index.ts`, for legacy resolvers that do not recognize `exports`. It should pretty much be identical to `packages/firebase/vertexai`. We're also going to have to make sure it builds by adding `ai` to the `components` array at the bottom of the file - see the rollup.config.js in this dir to see how it uses that array. I think that is the only place the components array is used, if it's used somewhere else we'll have to temporarily hardcode this `ai` build target into rollup.config.js instead.

Reply:
Which legacy resolvers do not recognize `exports`? I'd like to test that the new `packages/ai` directory I add fixes the problem.

Reply:
It looks like node >= 12 supports `exports` in `package.json`, and it looks like support for that was added in webpack 5 (https://webpack.js.org/blog/2020-10-10-webpack-5-release/#resolving). So i'll test with webpack 4.

Reply:
Done.

Tested without new `packages/firebase/ai` directory, and `ai` in `components`, and webpack 4 failed to resolve the import. After adding these, resolution succeeded.